tests/kafkatest/tests/core/security_rolling_upgrade_test.py

```diff
@@ -93,8 +93,8 @@ def roll_in_sasl_mechanism(self, security_protocol, new_sasl_mechanism):
         self.kafka.interbroker_sasl_mechanism = new_sasl_mechanism
         self.bounce()
 
-        # Bounce again with ACLs for new mechanism. Use old SimpleAclAuthorizer here to ensure that is also tested.
-        self.set_authorizer_and_bounce(security_protocol, security_protocol, KafkaService.SIMPLE_AUTHORIZER)
+        # Bounce again with ACLs for new mechanism.
+        self.set_authorizer_and_bounce(security_protocol, security_protocol, KafkaService.ACL_AUTHORIZER)
```

Comment:
Can invoke `self.set_authorizer_and_bounce(security_protocol, security_protocol)` as is done elsewhere in this file and eliminate the ability to specify a different authorizer class name in the method, like this:

```
    def set_authorizer_and_bounce(self, client_protocol, broker_protocol):
        self.kafka.authorizer_class_name = KafkaService.ACL_AUTHORIZER
```

Reply:
Thanks, updated.